### ISSUE TITLE
Automatically moving flashing window out of the way from the cursor

### ISSUE THREAD

**Issue by AllanChain** [AllanChain/sane-break#16]
See also #3.

Sometimes the flashing window can get into the way when you want to finish the work. It will be less disruptive if it can automatically move out of the way from the cursor.

In most cases, a window staying at the top center of the screen is okay. In addition, this can be tricky to implement. Therefore, low priority.

**Comment by wolftune** [comment]
although still nice visually, this is that much less necessary since the update that allows cursor clicking through the flashing area

**Comment by AllanChain** [comment]
You are right. I'm linking the commit (72056280f3b1e2775dd47b4cf4377fa2d88c3730) and closing this issue.
